changelog.d/pr-7257.bugfix

```diff
@@ -0,0 +1 @@
+Voice Broacast: The Now Playing Info Center now displays a voice broadcast instead of a voice message when a user is listening to a voice broadcast.
```

Comment:
```suggestion
Voice Broadcast: The Now Playing Info Center now displays a voice broadcast instead of a voice message when a user is listening to a voice broadcast.
```